examples/manifold/plot_lle_digits.py

```diff
@@ -235,8 +235,11 @@ def plot_embedding(X, title=None):
 # ----------------------------------------------------------------------
 # NCA projection of the digits dataset
 print("Computing NCA projection")
-nca = neighbors.NeighborhoodComponentsAnalysis(n_components=2, random_state=0)
+nca = neighbors.NeighborhoodComponentsAnalysis(init='random',
+                                               n_components=2, random_state=0)
 t0 = time()
+# X3 = X.copy()
+# X3.flat[::X.shape[1] + 1] += 0.01  # Make X invertible
```

Comment:
Please, either add this code (if it is necessary and it solves some issues) or remove these added lines. Commented code is really confusing for the reader, particularly in examples.

Reply:
I left these lines inadvertently. I am removing them